### ISSUE TITLE
Converting sequences to taxa names of 'otu_table' in phyloseq object

### ISSUE THREAD

**Issue by ardakeles28** [microbiome/microbiome#204]
Hi,

I am relatively new to bioinformatic analyses of 16S rRNA gene sequencing, and microbiome R package tutorials help me a lot. But I have an issue.

I am trying to explore the comparisons of microbiome community compositions. Since my 'otu_table' of the phyloseq object contains the sequences but not the taxa names, I am struggling to run most of the codes.

Can we convert the OTU sequences to the taxa names in the phyloseq object?

I really appreciate any help you can provide.

**Comment by antagomir** [comment]
Do you have mappings between sequence names and taxa names available?

**Comment by ardakeles28** [comment]
Yes, in the 'tax_table' of the phyloseq object, I have taxa names for the corresponding sequences.

But I suppose the majority of the codes require taxa names for the 'otu_table'. Because in the graphics of the tutorials I see taxa names such as 'Prevotalla oralis et rel.', which are found in the 'otu_table' of the example datasets.

**Comment by microsud** [comment]
You can find potential solutions here 
https://microbiome.github.io/tutorials/cleaning_taxonomy_table.html

**Comment by ardakeles28** [comment]
I think it solved the problem, thank you!

The thing is that, when I remove anything that is unclassified at the genus level for replacing the sequences as taxa names, the number of taxa decreased from 1162 to 114, which means the vast majority of my reads are unclassified at the genus level.

Would you suggest continuing the analysis with this number?

Appreciate your help.

**Comment by microsud** [comment]
It is common to get lower genus number then at OTU level because of aggregation.You should examine the tax_table to check how many OTUs are unclassified at Genus level. Example code below:
```
table(tax_table(pseq)[, "Genus"], exclude = NULL)
```
Then make your decision based on your research question.
I will close this issue for now.
